ci/ansible/roles/pulp-crane/tasks/main.yml

```diff
@@ -22,17 +35,42 @@
     state: true
     persistent: true
 
-- name: Install Crane configuration file
-  copy:
-    src: crane.conf
-    dest: /etc/crane.conf
-    owner: root
-    group: root
-    mode: 0644
+- block:
+  - name: Install Crane configuration file
+    copy:
+      src: crane.conf
+      dest: /etc/crane.conf
+      owner: root
+      group: root
+      mode: 0644
+
+  - name: Open firewall port for Crane (firewalld)
+    firewalld:
+      port: "5000/tcp"
+      state: enabled
+      permanent: true
+      immediate: true
+  when:
+    - not (ansible_distribution == "RedHat" and ansible_distribution_major_version|int == 6)
```

Comment:
> Also, if choosing the not equal approach maybe ansible_distribution == "RedHat" and ansible_distribution_major_version|int != 6 is more readable.

That would break this playbook on other distributions like CentOS 7. I think the best approach is to assume certain things by default, like that systemd and firewalld are in use, and then make exceptions for distros like RHEL 6 as necessary.